nvi-test/src/main/java/no/sikt/nva/nvi/test/TestUtils.java

```diff
@@ -60,7 +60,7 @@
 
 public final class TestUtils {
 
-    public static final int SCALE = 10;
+    public static final int SCALE = 4;
```

Comment:
Not relevant change, just set this to 4 because points in nvi have scale 4 after calculation.